backend/server.go

```diff
@@ -0,0 +1,82 @@
+package backend
+
+import (
+	"errors"
+	"net/http"
+	"os"
+	"sync"
+
+	"github.com/haya14busa/nintendo-switch-checker/nschecker"
+	"golang.org/x/net/context"
+	"google.golang.org/appengine"
+	"google.golang.org/appengine/log"
+	"google.golang.org/appengine/urlfetch"
+)
+
+func init() {
+	http.HandleFunc("/", handler)
+}
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	ctx := appengine.NewContext(r)
+	hc := urlfetch.Client(ctx)
+	var wg sync.WaitGroup
+	for _, s := range nschecker.Sources {
```

Comment:
Please filter sources here instead of splitting the sources.

It's exceptional cases for GAE and should be handled here.

Leaving some comments also will help.